DMX.Core.Api/Services/Orchestrations/ILabOrchestrationService.cs

```diff
@@ -14,5 +14,6 @@ public interface ILabOrchestrationService
         public ValueTask<Lab> AddLabAsync(Lab lab);
         public ValueTask<List<Lab>> RetrieveAllLabsAsync();
         public ValueTask<Lab> RemoveLabByIdAsync(Guid labId);
+        public ValueTask<Lab> RetrieveLabByIdAsync(Guid labId);

```

Comment:
Follow the CRUD operations order with these methods. So, our order should be:
`AddLabAsync`
`RetrieveAllLabs`
`RetrieveLabByIdAsync`
`RemoveLabByIdAsync`